app/src/main/java/com/example/myweatherapp/fragment/WeatherFragDetail.kt

```diff
@@ -0,0 +1,62 @@
+package com.example.myweatherapp.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import com.example.myweatherapp.data.entity.DaysWeather
+import com.example.myweatherapp.databinding.WeatherFragInDetailBinding
+import com.example.myweatherapp.mvp.contract.MyWeatherAppDetailContract
+import com.example.myweatherapp.mvp.model.MyWeatherAppDetailModel
+import com.example.myweatherapp.mvp.presenter.MyWeatherAppDetailPresenter
+import com.example.myweatherapp.mvp.view.MyWeatherAppDetailView
+import com.example.myweatherapp.utils.formatApp
+import com.example.myweatherapp.utils.formatJson
+
+class WeatherFragDetail : DialogFragment() {
```

Comment:
rename this to WeatherFragmentDetail pls

Reply:
Done thx